ecpy/utils/widgets/dict_editor.enaml

```diff
@@ -217,3 +224,42 @@ template DictEditor(Content):
             ed.iterable_name = 'pairs'
             ed.signal_name = 'pairs_changed'
             ed.operations << _make_ops(main.operations)
+
+
+enamldef FieldFieldEditor(Container):
+    """ Possible content for the Dictionary Editor containing a pair of
+    key-value Fields (String-like fields).
+
+    """
+
+    attr model
+    constraints = [hbox(k, v)]
+    padding = 1
+
+    Field: k:
+        text := model.key
+    Field: v:
+        text := model.value
+
+
+enamldef FieldFieldCompleterEditor(Container):
+    """ Possible content for the Dictionary Editor containing a pair of
+    key-value Fields (String-like fields) where the second one has the
+    line-completer that suggests possible entries from the database.
+
+    It expects the model to declare either the entries or entries_updater
+    function
+
+    """
+    attr model
+    constraints = [hbox(k, v)]
+    padding = 1
+
+    Field: k:
+        text := model.key
+    QtLineCompleter: v:
```

Comment:
Add query for the other parameters of the QtLineCompleter so that everything can be passed through the model